### PR TITLE
Documentation/license 3rdparty (#11553)

### PR DESCRIPTION
**Pull request by Simran-B**
* Add V8 gyp license
* Update license information, add date & missing ones
* Add remark to generateLicense.py (broken)

Backport of #11553

### CODE REVIEW

File: utils/node-module-licenses.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import csv, json, sys, os.path, re, io
+
+os.system("cd js/node && npx license-checker --relativeLicensePath --json > OUT")
+
+nv_re = re.compile(r'(@?[^@]*)@([^@]*)')
+git = "https://raw.githubusercontent.com/arangodb/arangodb/devel"
```
Comment (joerg84) [inline utils/node-module-licenses.py:7]: Should we make this configurable?

Comment (KVS85) [reply utils/node-module-licenses.py:7]: At least for current branch it makes sense to have `3.7` here. @Simran-B can you please confirm.

Comment (Simran-B) [reply utils/node-module-licenses.py:7]: This should be set to the respective branch name indeed (here: 3.7)

File: utils/node-module-licenses.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import csv, json, sys, os.path, re, io
+
+os.system("cd js/node && npx license-checker --relativeLicensePath --json > OUT")
```
[the remark this answers is not in the document]
Comment (fceller) [reply utils/node-module-licenses.py:4]: we only use this internal

**Pull request merged**